x-pack/plugins/alerting/server/rules_client/rules_client.ts

```diff
@@ -1083,10 +1088,86 @@ export class RulesClient {
     }
   }
 
+  public async getActionErrorLogWithAuth({
```

Comment:
lets make sure this has the same level of testing as the other `WithAuth` handlers in the rules client :slightly_smiling_face: